webdriver/support/fixtures.py

```diff
@@ -115,3 +117,83 @@ def inner(path, query="", fragment=""):
                                   fragment))
         return rv
     return inner
+
+def switch_to_inactive(session):
+    """Create a new browsing context, switch to that new context, and then
+    delete the context."""
+    def switch_to_inactive():
+        initial = session.send_session_command("GET", "window/handles")
+
+        session.send_session_command("POST",
+                                     "execute/sync",
+                                     dict(script="window.open();", args=[]))
```

Comment:
I'm happy to use the higher-level function here (and in other set-up/tear-down logic), and I will update the patch accordingly. As to "why,"--the use of lower-level functions for set-up/tear-down logic in this patch is largely a vestige of my thinking prior to the design discussions we've had since February.